### PR TITLE
fix Typo in GSIK

### PR DESCRIPTION
**Pull request by kinichiro**
reported by @ClarkInAz by #441

- section 6.1, step 3, Passive -> Power input
- section 3.1, step 32, pin 2 -> pin 1
- section 4.1, step 11, add word "layer" and screen shot
- section 3.1, step 4, and section 4.1 step 2,
  -> Set the appropriate 'paper size' ('A4','8.5x11' etc.)

### CODE REVIEW

File: src/getting_started_in_kicad/getting_started_in_kicad.adoc [hunk 1]
```diff
@@ -829,13 +831,15 @@ image::images/gsik_tutorial1_080.png[gsik_tutorial1_080_png]
     preferred to actual wires because they make the schematic much
     less messy.
 
-11. Now we will define the edge of the PCB. Select 'Edge.Cuts' from
+11. Now we will define the edge of the PCB. Select 'Edge.Cuts' layer from
```
Comment (nickoe) [inline src/getting_started_in_kicad/getting_started_in_kicad.adoc]: shouldn't this be  _Select the 'Edge.Cuts' layer from_

**Comment by kinichiro** [comment]
@nickoe

>shouldn't this be Select the 'Edge.Cuts' layer from

I can't see what is wrong.
Could you give me the way to fix this, please ?
I should not add 'layer' and should remove it ?

File: src/getting_started_in_kicad/getting_started_in_kicad.adoc (hunk 1 above)
Comment (ciampix) [inline src/getting_started_in_kicad/getting_started_in_kicad.adoc]: I agree. As it was it is not really wrong but adding the word "layer" is more clear. IMHO of course.

Comment (ciampix) [inline src/getting_started_in_kicad/getting_started_in_kicad.adoc]: Yes: Select the 'Edge.Cuts' layer from ...

**Comment by kinichiro** [comment]
I see. I fixed and pushed the change.
'Select the Edge.Cuts layer from ...'

**Pull request merged**